sdk/core/Azure.Core/tests/TestFramework/RecordedTestBase.cs

```diff
@@ -17,6 +17,13 @@ public abstract class RecordedTestBase : ClientTestBase
 
         public RecordedTestMode Mode { get; }
 
+        /// <summary>
+        /// Flag you can (temporarily) enable to save failed test recordings
+        /// and debug/re-run at the point of failure without re-running
+        /// potentially lengthy live tests.  This should never be checked in.
```

Comment:
Partly to avoid restarting VS but also partly to make it clear when you're doing something dangerous.  I'd worry about leaving that env var on by accident.